src/platform-includes/session-replay/setup/javascript.capacitor.mdx

```diff
@@ -1,15 +1,13 @@
 Several options are supported and passable using the integration constructor. See the [configuration documentation](/platforms/javascript/guides/capacitor/session-replay/configuration/) for more details. To set up the integration, add the following to your Sentry initialization:
 
-<Note>
-The minimal supported version of Sentry's SDK for Capacitor is 0.11.1.
-</Note>
+<Note>The minimal supported version of Sentry's SDK for Capacitor is 0.11.1.</Note>
```

Comment:
Does this work okay like this?

Our internal docs on docs say notes have to be formatted like this to work with Gatsby:

```suggestion
<Note>

The minimal supported version of Sentry's SDK for Capacitor is 0.11.1.

</Note>
```

Reply:
In this scenario either way is fine. since there's no markdown in the note.

If there was then it would not work since the markdown would not be parsed.